src/analytics/Events.tsx

```diff
@@ -705,4 +705,7 @@ export enum EarnEvents {
   earn_info_learn_press = 'earn_info_learn_press',
   earn_info_earn_press = 'earn_info_earn_press',
   earn_active_pools_cta_press = 'earn_active_pools_cta_press',
+  earn_home_learn_more_press = 'earn_home_learn_more_press',
+  earn_multi_pool_add_to_pool_press = 'earn_multi_pool_add_to_pool_press',
+  earn_multi_pool_exit_pool_press = 'earn_multi_pool_exit_pool_press',
```

Comment:
Can we just reuse the existing events for the single pool card instead of more new events w/ additional props you're adding below?  Guess when we launch, these might also be called deposit / withdraw at the time we launch, so we could name it something more generic if we want new events.

Reply:
I didn't re-use the old events because they use `EarnCommonProperties` which have 'aave-v3' hardcoded, and I would've needed to add the `poolId` prop which I think we would want to know which pool it was pressed for. I also don't think there is an event for first time adding to pool since in the single-pool case there is a cta press that takes you right to enter amount.

I can try and re-use `earn_deposit_more_press` and `earn_exit_pool_press` and modify `EarnCommonProperties` and then just have a new event for adding to an open pool, or I can just update these names to be more generic and add any extra props that we would want.

Reply:
eventually `EarnCommonProperties` would have to be modified to work with multiple pools, so don't see that as in issue. Actually, seems cleaner to start with new events since this is a new screen, but would be nice if we can just change names to be more generic, as it looks like we're gonna replace the "Add to pool" / "Exit pool" naming soon with the designs presented today. Maybe we can do `earn_pool_card_cta_press` with prop for deposit / withdraw + EarnCommonProperties. wdyt?

Reply:
yeah I like that idea of just the single event. I think it would also make sense to have a balance property, that way we have that info for the withdraw event, and also it can let us differentiate between a user adding to a pool for the first time vs. depositing more